subprojects/test-runner/kubernetes/src/main/kotlin/com/avito/k8s/KubernetesApiImpl.kt

```diff
@@ -13,6 +13,7 @@ import kotlinx.coroutines.delay
 internal class KubernetesApiImpl(
     private val kubernetesClient: KubernetesClient,
     loggerFactory: LoggerFactory,
+    override val needForward: Boolean,
```

Comment:
We should change bool to sealed class PodPortForwardConfig { disabled, Enabled(port = 5555)